app/models/container_deployment_node.rb

```diff
@@ -0,0 +1,6 @@
+class ContainerDeploymentNode < ApplicationRecord
+  belongs_to :vm
+  belongs_to :container_deployment
+  serialize :labels, Hash
```

Comment:
Looks like there's some [precedence for `:serialize Hash`](https://github.com/ManageIQ/manageiq/blob/1e827d0ffa8bc2ac2d0cdb731ac3f1f703a4ace3/app/models/host.rb#L90).